Http/Controllers/ItkPrometheusController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Modules\ItkPrometheus\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Routing\Controller;
+use Modules\ItkPrometheus\Service\PrometheusService;
+
+class ItkPrometheusController extends Controller
+{
+    private PrometheusService $service;
+
+    public function __construct(PrometheusService $service)
+    {
+      $this->service = $service;
+    }
+
+  /**
+     * Display a listing of the resource.
+     * @return Response
+     */
+    public function index()
+    {
+        return view('itkprometheus::index');
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     * @return Response
+     */
+    public function create()
+    {
+        return view('itkprometheus::create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     * @param  Request $request
+     * @return Response
```

Comment:
Normally we have an empty line between the different doc block types. But I do not known what the coding standard for freescout is.